GitUI/HelperDialogs/FormResetAnotherBranch.cs

```diff
@@ -96,7 +97,7 @@ private void Ok_Click(object sender, EventArgs e)
                 return;
             }
 
-            GitExtUtils.ArgumentString command = Commands.PushLocal(gitRefToReset.CompleteName, _revision.ObjectId, Module.WorkingDir, Module.GetPathForGitExecution, ForceReset.Checked);
+            GitExtUtils.ArgumentString command = Commands.PushLocal(gitRefToReset.CompleteName, _revision.ObjectId, Module.WorkingDir.ToPosixPath(), Module.GetPathForGitExecution, ForceReset.Checked);
```

Comment:
No, rather handle it correctly in `PushLocal`. It shall call `getPathForGitExecution` before checking for `NativeDirectorySeparatorChar`.